addons/account/models/account_reconcile_model.py

```diff
@@ -461,20 +461,14 @@ def _prepare_reconciliation(self, st_line, aml_ids=[], partner=None):
         return lines_vals_list + writeoff_vals_list
 
     def _prepare_widget_writeoff_vals(self, st_line_id, write_off_vals):
-        fixed_write_off_vals = dict(write_off_vals, currency_id=st_line_id.company_id.currency_id.id)
-        counterpart_vals = st_line_id._prepare_counterpart_move_line_vals(fixed_write_off_vals)
-
+        counterpart_vals = st_line_id._prepare_counterpart_move_line_vals({
+                **write_off_vals,
+                'currency_id': st_line_id.company_id.currency_id.id,
+            })
         return {
-            'name': counterpart_vals['name'],
+            **counterpart_vals,
             'balance': counterpart_vals['amount_currency'],
-            'debit': counterpart_vals['debit'],
-            'credit': counterpart_vals['credit'],
```

Comment:
@smetl So it is deliberate that `amount_currency == debit - credit`?

Reply:
@william-andre Yes.. It's because both widgets (manual + bank) are using the same common way to use the reconciliation models. This has been fixed many times and the result is that